go/protocol/keybase1/user.go

```diff
@@ -144,6 +144,7 @@ type ListTrackingJSONArg struct {
 type SearchArg struct {
 	SessionID int    `codec:"sessionID" json:"sessionID"`
 	Query     string `codec:"query" json:"query"`
+	Service   string `codec:"service" json:"service"`
```

Comment:
Yes, all of that is generated code. You'll want to change [the AVDL file it came from](https://github.com/keybase/client/blob/master/protocol/avdl/keybase1/user.avdl), and rerun `make` in `./protocol/`.